.github/workflows/main.yml

```diff
@@ -17,12 +17,15 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 14.x
+          node-version: 16.x
```

Comment:
I'd go with 18.x, if we only pick one, but 16 is good for now as well

Reply:
I'm more than happy to go with 18.x, will update tomorrow :)